docs/dev/table/connect.md

```diff
@@ -1094,6 +1094,23 @@ For append-only queries, the connector can also operate in [append mode](#update
 The connector can be defined as follows:
 
 <div class="codetabs" markdown="1">
+<div data-lang="Java/Scala" markdown="1">
+{% highlight java %}
+.connect(
+  new HBase()
+    .version("1.4.3")                      // required: valid connector versions are "1.4.3"
+    .tableName("hbase_table_name")         // required: HBase table to connect to
+    .zookeeperQuorum("localhost:2181")     // required: Zookeeper quorum address of HBase
+    .zookeeperNodeParent("/test")          // optional: zookeeper node parent path of HBase
+    .writeBufferFlushMaxSize("10mb")       // optional: threshold when to flush buffered request based on
+                                           // the memory byte size of rows currently added.
+    .writeBufferFlushMaxRows(1000)         // optional: threshold when to flush buffered request based on
+                                           // the number of rows currently added.
+    .writeBufferFlushInterval("2s")        // optional: an interval when to flushing buffered requesting
+                                           // if the interval passes, in milliseconds.
```

Comment:
I would like to keep the same comments as the comments in DDL tab.

Reply:
Fine, It should be.